### PR TITLE
(#101) Added gitignore instructions and samples

### PR DESCRIPTION
**Pull request by adrianhall**
Added instruction in README.md to create a gitignore file (reference gitignore.io) and added gitignore files to the two samples.

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@adrianhall__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

File: README.md [hunk 1]
```diff
@@ -48,6 +48,9 @@ An official npm package will be published soon.
 
   `npm install express azure-mobile-apps --save`
 
+0. Create a suitable .gitignore file.  You can generate a suitable .gitignore
+file using the generator at [gitignore.io](https://www.gitignore.io)
+
```
Comment (danderson00) [inline README.md]: .gitignore just needs to contain node_modules

Comment (adrianhall) [inline README.md]: For samples and in general, yes.  However, if the developer is doing other stuff, there may be other generated files.  Better to point them to standardized gitignore stuff than to assume node_modules is the only thing.

**Comment by danderson00** [comment]
Resolves https://github.com/Azure/azure-mobile-apps-node/issues/101

**Pull request merged**